docs/tutorial.py

```diff
@@ -0,0 +1,7 @@
+import pyhf_combine_converter as converter
+
+##convert to pyhf
+converter.pyhf_converted_from_datacard.main("converted_datacard.txt", outfile = "pyhf_workspace.json")
+##convert back to datacard
+converter.pyhf_convert_to_datacard.main("pyhf_workspace.json", outdatacard = "combine_datacard.txt", shapefile = "shapes.root")
+
```

Comment:
```suggestion
from pyhf_combine_converter import pyhf_converted_from_datacard
from pyhf_combine_converter import pyhf_convert_to_datacard

##convert to pyhf
pyhf_converted_from_datacard.main(
    "converted_datacard.txt", outfile="pyhf_workspace.json"
)
##convert back to datacard
pyhf_convert_to_datacard.main(
    "pyhf_workspace.json", outdatacard="combine_datacard.txt", shapefile="shapes.root"
)
```

As no methods exist at import of `pyhf_combine_converter` (like Alex's point above) then need to import the files. This suggestion also runs Black on the file.

There's a problem here though as the Python API isn't working as expected.

```console
[cmsusr@e619bd4fd100 Pyhf-to-Combine-converter]$ python3 docs/tutorial.py 
/home/cmsusr/.local/lib/python3.8/site-packages/numpy/core/getlimits.py:499: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/cmsusr/.local/lib/python3.8/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
/home/cmsusr/.local/lib/python3.8/site-packages/numpy/core/getlimits.py:499: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/cmsusr/.local/lib/python3.8/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
Traceback (most recent call last):
  File "docs/tutorial.py", line 5, in <module>
    pyhf_converted_from_datacard.main(
TypeError: main() got an unexpected keyword argument 'outfile'
```

I forgot about this, but I think that what will be needed is to to refactor things so that:
* The `main` functions get renamed (e.g. Alex's suggestion of `pyhf_convert_to_datacard`)
* The `parser = OptionParser` stuff gets moved to the

```python
if __name__ == "__main__":
    main()
```

block and if needed things can be put inside a `main` function still. This separates out the Python API from the CLI API. This got broken when I was making the CLI API in PR #19.

This should probably get fixed in another PR before this goes in.